src/pages/Home.vue

```diff
@@ -99,6 +99,7 @@ export default {
     return {
       tableColumns: ["name", "number_of_viewers"], // columns for the table
       tableData: [],
+      uniqueUsersList: [], // holds the number of unique users for all the plios fetched
```

Comment:
Can change the name a little?
Feels like this array contains the unique users, not the number of unique users
Maybe `numberOfUniqueUsersList` or `countUniqueUsersList`
And i think we can skip the `list` waala part in the name